### PR TITLE
chore: ruff linters

### PR DESCRIPTION
**Pull request by nodegard**
## Description
Please describe the change you have made.

## Checklist:
- [ ] Tests added/updated.
- [ ] Documentation updated.
- [ ] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/power-ops-sdk/blob/master/CHANGELOG.md).
- [ ] Version bumped. If triggering a new release is desired, bump the version number in [pyproject.toml](https://github.com/cognitedata/power-ops-sdk/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: cognite/powerops/resync/config/_main.py [hunk 1]
```diff
@@ -197,9 +201,10 @@ def validate_shop_related_files(cls, value):
 
     @classmethod
     def instantiate_from_dict(cls, config: dict) -> CogShopConfig:
-        time_series_mappings_v2 = []
-        for time_series_mapping in config["time_series_mappings_v2"]:
-            time_series_mappings_v2.append(TimeSeriesMappingV2.load_from_dict(time_series_mapping))
```
Comment (notrixbe) [inline cognite/powerops/resync/config/_main.py:202]: mapping removed ?

File: cognite/powerops/resync/config/production/plant.py [hunk 2]
```diff
@@ -77,31 +78,33 @@ def from_cdf_resources(cls, asset: Asset, relationships: list[Relationship], **k
...
+            elif RelationshipLabel.FEEDING_FEE_TIME_SERIES in labels:
                 plant.feeding_fee_time_series = ts_ext_id
-            elif rl.P_MIN_TIME_SERIES in labels:
+            elif RelationshipLabel.P_MIN_TIME_SERIES in labels:
                 plant.p_min_time_series = ts_ext_id
-            elif rl.P_MAX_TIME_SERIES in labels:
+            elif RelationshipLabel.P_MAX_TIME_SERIES in labels:
                 plant.p_max_time_series = ts_ext_id
-            elif rl.HEAD_DIRECT_TIME_SERIES in labels:
+            elif RelationshipLabel.HEAD_DIRECT_TIME_SERIES in labels:
```
Comment (notrixbe) [inline cognite/powerops/resync/config/production/plant.py:93]: 👍

File: cognite/powerops/resync/config/_main.py (hunk 1 above)
Comment (nodegard) [reply cognite/powerops/resync/config/_main.py:202]: just rewrote the for loop into a list comprehension

**Review by notrixbe — APPROVED** [review]
great! 
think only 1 thing to check

**Pull request merged**